src/edge_kernels/MomentumABLWallShearStressEdgeKernel.C

```diff
@@ -25,27 +25,37 @@ namespace nalu {
 
 template<typename BcAlgTraits>
 MomentumABLWallShearStressEdgeKernel<BcAlgTraits>::MomentumABLWallShearStressEdgeKernel(
+  bool slip,
   stk::mesh::MetaData& meta,
-  ElemDataRequests& faceDataPreReqs
-) : NGPKernel<MomentumABLWallShearStressEdgeKernel<BcAlgTraits>>(),
+  std::string coordname,
+  ElemDataRequests& faceDataPreReqs,
+  ElemDataRequests& elemData
+  ) : NGPKernel<MomentumABLWallShearStressEdgeKernel<BcAlgTraits>>(),
+    slip_(slip),
+    coordinates_(get_field_ordinal(meta, coordname)),
     exposedAreaVec_(get_field_ordinal(meta, "exposed_area_vector", meta.side_rank())),
     wallShearStress_(get_field_ordinal(meta, "wall_shear_stress_bip", meta.side_rank())),
-    meFC_(sierra::nalu::MasterElementRepo::get_surface_master_element<BcAlgTraits>())
+    meFC_(MasterElementRepo::get_surface_master_element<typename BcAlgTraits::FaceTraits>()),
+    meSCS_(MasterElementRepo::get_surface_master_element<typename BcAlgTraits::ElemTraits>())
 {
   faceDataPreReqs.add_cvfem_face_me(meFC_);
+  elemData.add_cvfem_surface_me(meSCS_);
 
   faceDataPreReqs.add_face_field(exposedAreaVec_, BcAlgTraits::numFaceIp_, BcAlgTraits::nDim_);
   faceDataPreReqs.add_face_field(wallShearStress_, BcAlgTraits::numFaceIp_, BcAlgTraits::nDim_);
+
+  elemData.add_coordinates_field(coordinates_, BcAlgTraits::nDim_, CURRENT_COORDINATES);
```

Comment:
It looks like coordinates is not used in the algorithm. Can eliminate this registration step, as well as tracking coordinates for this kernel.